wandb/sdk_py27/wandb_init.py

```diff
@@ -364,8 +365,14 @@ def init(self):
             run._set_run_obj_offline(run_proto)
         else:
             ret = backend.interface.communicate_check_version()
-            if ret and ret.message:
-                wandb.termlog(ret.message)
+            if ret and ret.upgrade_message:
```

Comment:
while it is true that we should always have an upgrade message if the package was deleted or yanked. lets not encode this logic here, let's treat them as separate.
if ret:
   if ret.upgrade_message:
     ...
   if ret.yank_message:
       ....